RecoTauTag/RecoTau/interface/PFRecoTauClusterVariables.h

```diff
@@ -0,0 +1,165 @@
+
+/** \class PFRecoTauClusterVariables
+ *                                                                                                                                                                                   
+ * Cluster variables to be used in MVA tau ID
+ *                                                                                                                                                                                   
+ * \author Aruna Nayak, DESY
+ *                                                                                                                                                                                   
+ */
+
+#include "DataFormats/Candidate/interface/Candidate.h"
+#include "DataFormats/TauReco/interface/PFTau.h"
+#include "DataFormats/TauReco/interface/PFTauFwd.h"
+#include "DataFormats/Math/interface/deltaR.h"
+#include <TMath.h>
+
+using namespace reco;
+
+namespace
+{
+
+  const std::vector<reco::PFCandidatePtr>& getPFGammas(const reco::PFTau& tau, bool signal = true) {
+    if (signal)
+      return tau.signalPFGammaCands();
+    return tau.isolationPFGammaCands();
+  }
+
+  bool isInside(float photon_pt, float deta, float dphi){
+
+    if(photon_pt==0){return false;}
+
+    if(
+       (dphi < TMath::Min(0.3, TMath::Max(0.05, 0.352476*TMath::Power(photon_pt, -0.707716)))) && \
+       (deta < TMath::Min(0.15, TMath::Max(0.05, 0.197077*TMath::Power(photon_pt, -0.658701))))
```

Comment:
Please no magic numbers. Set well-named constants to these values and then use them in place of the magic numbers.